<?xml version="1.0"?>
<launch>
    <param name="use_sim_time" value="true"/>

    <node pkg="rosbag" type="play" name="rosbag"
          args="--clock $(find gp_occ_mapping)/../data_set/stdr/stdr_data.bag">

        <remap from="/robot0/laser_0" to="/scan"/>
    </node>

    <!-- <node pkg="rviz" type="rviz" name="rviz"/> -->

</launch>